lib/features/debug/screens/debug_screen/debug_screen_widget_model.dart

```diff
@@ -70,9 +70,12 @@ class DebugScreenWidgetModel extends WidgetModel<DebugScreen, DebugScreenModel>
   @override
   void initWidgetModel() {
     super.initWidgetModel();
+    _urlState = ValueNotifier<UrlType>(
+      _resolveUrlType(model.configNotifier.value.url),
+    );
+    _themeState = ValueNotifier<ThemeMode>(model.currentThemeMode.value);
```

Comment:
is it required to initialize it here? can we set it as initial value?

Reply:
Unfortunately no, because we cannot initialize a variable with a value from the model